Documentation/articles/getting_started/getting_started.md

```diff
@@ -1,10 +1,30 @@
 # Getting Started
 
-This section walks you through the basics of MonoGame and help you create your first game.
+This section walks you through the basics of MonoGame and helps you create your first game.
+
+First, select the toolset and operating system you will be working with to create your first MonoGame project and then continue your journey to understand the basic layout of a MonoGame project.
+
+By the end of this tutorial set, you will have a working project to start building from for your target platform and be ready to tackle your next steps.
+
+## Creating a new Project
+
+- 3.8
```

Comment:
These docs are all for 3.8... lets remove references to 3.8 and move the docs out of a 3.8 folders in the links below.

Reply:
3.7 and all references to it be gone.